src/main/java/seedu/storemando/ui/MainWindow.java

```diff
@@ -137,14 +138,15 @@ private void setWindowDefaultSize(GuiSettings guiSettings) {
     }
 
     /**
-     * Opens the help window or focuses on it if it's already opened.
+     * Opens storemando's user guide on user's browser if user's desktop supports it.
+     * Otherwise, opens the help window or focuses on it if it's already opened.
      */
     @FXML
     public void handleHelp() {
-        if (!helpWindow.isShowing()) {
-            helpWindow.show();
+        if (BrowserUtil.isBrowserAccessible() && BrowserUtil.isConnectedToInternet()) {
+            BrowserUtil.displayWebsite(HelpWindow.USERGUIDE_URL);
```

Comment:
This may be a violation of Law of Demeter, cause browserutil calls a method that calls helpwindow's string. Maybe can consider shifting the USERGUIDE_URL to BrowserUtil instead :)

Reply:
I think it is still fine as the class MainWindow has helpwindow as the immediate neighbour and I'm passing the URL as an argument to the displayWebsite method in BrowserUtil.

Reply:
Upon further thinking, I have shifted it as per your suggestion as it was more practical. Thanks and fixed!